testing/test_cli.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+
+def find_tables(conn):
+    """return a list of tables"""
+    with conn.cursor() as cur:
+        cur.execute(
+            "select table_name from information_schema.tables where table_schema='public'"
+        )
+        return cur.fetchall()
+
+
+def test_createtables(conn):
+    err = os.system("ethindex createtables")
+    assert err == 0
+    tables = find_tables(conn)
+    print("FOUND", tables)
+    assert len(tables) >= 3
+
+
+def test_droptables_without_force(conn):
+    """droptables require the user to pass --force"""
+    err = os.system("ethindex createtables")
+    assert err == 0
+    tables_before = find_tables(conn)
+    err = os.system("ethindex droptables")
+    assert err == 256  # exit code 1
```

Comment:
wait what?
can we use something else that we get the actual return code?

Reply:
I think we should use the subprocess module.

Reply:
using subprocess.call now